### ISSUE TITLE
training hyper-parameters for ablation studies

### ISSUE THREAD

**Issue by ethanhe42** [deepseek-ai/DeepSeek-V3#489]
Thanks for the great work. Could you share the training hyper-parameters for 16B and 236B ablation studies? specifically learning rate schedule, batch size schedule, maximum sequence length, bias update speed, etc.
